site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -184,6 +184,8 @@ Commands can be sent following a *request/response* pattern or being *one-way*.
 
 For *Request/Response* commands, devices send their responses to commands by means of sending an MQTT *PUBLISH* message to a topic that is specific to the command that has been executed. The MQTT adapter accepts responses being published using either QoS 0 or QoS 1.
 
+The MQTT adapter checks the configured [message limit] ({{< relref "/concepts/resource-limits.md" >}}) before accepting any command requests and responses. In case of incoming command requests from business applications, if the message limit is exceeded, the Adapter rejects the message with the reason `amqp:resource-limit-exceeded`. And for the incoming command responses from devices, the Adapter rejects the message and closes the connection to the client. 
```

Comment:
I think this is a duplication of the *Message Limits* section above. Apart from that, it seems to still contain wording that is specific to the AMQP adapter ...

Reply:
_Message Limits_ in the top section gives only a short description of message limits. In addition to that, each section corresponding to `telemetry` and `event` contain information regarding the outcome of the message limit exceeded e.g _the message is discarded and the MQTT connection is closed_. In this PR, it has been updated for the _commands_ section.

> Apart from that, it seems to still contain wording that is specific to the AMQP adapter ...

In this part, the outcome of the command request sent by the north bound applications is described with the reason for rejection as `amqp:resource-limit-exceeded`. It is followed by the outcome of the command response to devices in case of limit exceeded.

Reply:
I see :+1: